src/tests/class_tests/openms/source/IDMergerAlgorithm_test.cpp

```diff
@@ -38,6 +38,8 @@
 #include <OpenMS/FORMAT/IdXMLFile.h>
 #include <OpenMS/SYSTEM/StopWatch.h>
 #include <OpenMS/test_config.h>
+#include <QFile>
```

Comment:
remnants from testing? Not used?

Reply:
wow quick spotting. yes, found it as well.